.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -43,3 +43,29 @@ Relevant software versions:
 ## Additional context
 
 <!-- Add any other context about the problem here, such as stack traces or debugging info. -->
+
+#### Definition of Done
+
+- [ ] This bug is **done**:
+  - [ ] Bug resolved to **user's** satisfaction
+  - [ ] Automated tests are passing
+  - [ ] Code is peer reviewed and pushed to master
+  - [ ] Deployed successfully to test environment
+  - [ ] Checked against CODING guidelines
+  - [ ] Relevant new tasks are added to backlog and communicated to the team
+  - [ ] Important decisions recorded in the issue ticket
+  - [ ] Sprint board is updated
+  - [ ] Readme/Changelog/Diagrams are updated
+  - [ ] Product Owner has approved as complete
+  - [ ] Meets non-functional requirements:
```

Comment:
I think all those non-functional requirements are more relevant to new features rather than to bugs. For bugs I would just check they don't introduce any regressions.

Reply:
Fair enough. Will change that.

Reply:
I added:

> No regression to functional or [non-functional](https://github.com/linz/geostore/blob/master/.github/ISSUE_TEMPLATE/user_story.md) requirements

Although, I wonder if the requirement to pass automated tests should cover this? Maybe not in all cases though?